manifests/p/PlayStation/PSPlus/12.1.0/PlayStation.PSPlus.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: PlayStation.PSPlus
+PackageVersion: 12.1.0
+PackageLocale: en-US
+Publisher: Sony Interactive Entertainment Inc.
+PackageName: PlayStation Plus

```

```suggestion
PackageName: PlayStationPlus
```
This should match Control Panel Exactly